src/pages/copylist/CopyListPage.module.css

```diff
@@ -1,3 +0,0 @@
-.queryBuilderButton {
```

Comment:
This was no longer needed as `CopyListPage` now reuses the button rendering logic inside `EditListResultViewer`